install/init.go

```diff
@@ -165,4 +165,14 @@ func (s *SealosInstaller) SendKubeConfigs(masters []string, isMaster0 bool) {
 	SendPackage(cert.SealosConfigDir+"/admin.conf", masters, cert.KubernetesDir, nil, nil)
 	SendPackage(cert.SealosConfigDir+"/controller-manager.conf", masters, cert.KubernetesDir, nil, nil)
 	SendPackage(cert.SealosConfigDir+"/scheduler.conf", masters, cert.KubernetesDir, nil, nil)
+
+	// fix > 1.19.1 kube-controller-manager and kube-scheduler use the LocalAPIEndpoint instead of the ControlPlaneEndpoint.
+	if VersionToIntAll(Version) >= 1191 {
```

Comment:
when fix the localControPlaneEndpoint ,  1.19.3 fix. 
 change to `1193 => VersionToIntAll(Version) >= 1191`